index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<style>
+    img {
+        display: block;
+        max_width: 100%;
+
+
+    }
+
+    #image {
+        max-width: 100%;
+        width: 100%;
+    }
+</style>
```

Comment:
Kindly move your inline style to your `style.css` to make your code looks simplified and professional👌